docs/non_containerized_NSFS.md

```diff
@@ -467,13 +467,32 @@ sudo node src/cmd/manage_nsfs bucket delete --config_root ../standalon/config_ro
 sudo node src/cmd/manage_nsfs bucket list --config_root ../standalon/config_root 2>/dev/null
 
 ```
+NSFS management CLI command will create both account and bucket dir if it's missing in the config_root path.
```

Comment:
```suggestion
NSFS management CLI run will create both accounts, access_keys, and buckets directories if they are missing under the config_root directory.
```

Reply:
I just moved this.
I will change it according to your suggestion, thanks.